test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -238,7 +247,8 @@ TEST(help_page_printing, full_information)
                "example\n"
                "example2" +
                basic_version_str;
-    EXPECT_TRUE(ranges::equal((std_cout | std::view::filter(!is_space)), expected | std::view::filter(!is_space)));
+    EXPECT_TRUE(ranges::equal((std_cout | std::view::filter(!is_space)), expected | std::view::filter(!is_space))) <<
+      "\nexpected:\n----\n" << expected << "\n\ncurrent:\n----\n" << std_cout;
```

Comment:
indentation?

Reply:
I'll just remove this again. It was just for debugging.